packages/eslint-plugin-orbit-components/__tests__/themeRTL.test.ts

```diff
@@ -0,0 +1,75 @@
+import ruleTester from "../ruleTester";
+import themeRTL, { ERROR } from "../src/rules/themeRTL";
+
+describe("theme-rtl", () => {
+  // @ts-expect-error TODO
+  ruleTester.run("theme-rtl", themeRTL, {
+    valid: [
+      {
+        code: `
+          import styled from 'styled-components';
+          import { right } from "@kiwicom/orbit-components/lib/utils/rtl";
+
+            const StyledWrapper = styled.div\`
+              \${right}: 0;
+            \`
+          `,
+      },
+      {
+        code: `
+          import styled from 'styled-components';
+          import { right } from "@kiwicom/orbit-components/lib/utils/rtl";
+
+            const StyledWrapper = styled.div\`
+              margin-\${right}: \${({ theme }) => theme.orbit.spaceSmall};
+            \`
+          `,
+      },
+      {
+        code: `
+          import styled from 'styled-components';
+          import { right } from "@kiwicom/orbit-components/lib/utils/rtl";
+
+            const StyledWrapper = styled.div\`
+              margin-\${right}: \${({ theme }) => theme.orbit.spaceSmall};
+            \`
+          `,
+      },
+    ],
+    invalid: [
```

Comment:
Would also test a case when the conditional expression returns different spacing value, e.g.:

```js
padding: ${({ theme }) => theme.rtl ? `0 ${theme.orbit.spaceSmall} 0 ${theme.orbit.spaceXSmall}` : `0 ${theme.orbit.spaceXSmall} 0 ${theme.orbit.spaceSmall}`};
```

This is a case that should be covered by use of `rtlSpacing` not `right/left` directly.